src/components/Icons/Bars.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Bars = (props) => {
+  return (
+    <svg viewBox="0 0 24 24" aria-hidden="true" {...props}>
```

Comment:
If you are using font awesome icons you can use an icon from it instead (:
I don't think there's a reason to make each different icon into a different component. What you can do is create a reusable icon button component that gets the icon through props.